wolfcrypt/src/pkcs12.c

```diff
@@ -1089,13 +1137,843 @@ int wc_PKCS12_parse(WC_PKCS12* pkcs12, const char* psw,
             buf = NULL;
         }
 
-        freeCertList(certList, pkcs12->heap);
+        wc_FreeCertList(certList, pkcs12->heap);
     }
 
     return ret;
 }
 
 
+
+static int wc_PKCS12_shroud_key(WC_PKCS12* pkcs12, WC_RNG* rng,
```

Comment:
What do the return codes mean for a caller?  And the next several functions too?